### PR TITLE
Upgrade Android API level to 31

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
             android:theme="@style/AppTheme.NoActionBarLaunch"
             android:launchMode="singleTask"
             android:screenOrientation="portrait"
+            android:exported="true"
```
Comment (yksolanki9) [inline android/app/src/main/AndroidManifest.xml:21]: Why do we need this?

**Review by yksolanki9 — APPROVED** [review]

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Julias0) [reply android/app/src/main/AndroidManifest.xml:21]: +1

**Review by Julias0 — CHANGES_REQUESTED** [review]
1 question

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Dimple16) [reply android/app/src/main/AndroidManifest.xml:21]: @Julias0 
We need to explicitly set this for app targeting android 12 and above. adding references:
https://developer.android.com/guide/topics/manifest/activity-element#exported
https://developer.android.com/about/versions/12/behavior-changes-12
https://stackoverflow.com/questions/68554294/androidexported-needs-to-be-explicitly-specified-for-activity-apps-targeting

**Review by Julias0 — APPROVED** [review]

**Pull request merged**